### PR TITLE
WIP: Documentation pulled from the discussion in #642

### PR DESCRIPTION
**Pull request by btellstrom**
Preliminary documentation based on the comments and discussion in issue #642.

### CODE REVIEW

File: doc/Scripts/script_notes.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Scripts
+## launch_repairnator.sh
+The launch repairnator has two options:
```
Comment (surli) [inline doc/Scripts/script_notes.md]: Actually there is only one option which is optional: the list of build ids

File: doc/Scripts/script_notes.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+```
+./launch_repairnator.sh
```
Comment (surli) [inline doc/Scripts/script_notes.md]: maybe, put the standard usage: `./launch_repairnator.sh [list_of_buildIDS.txt]`
In usages `[]` are expressing an optional argument

File: doc/Dockerpool/SkipDeleteArchitectureNotes.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+All scripts that are using the pipeline are using a docker-container
+created from a docker image of repairnator-pipeline. This implies that
+we only use pipeline through Docker. *Is this hardcoded, or just how we
+do it? Considering the skipDelete is a part of the repairnator config
+I'd assume it is hardcoded, but that perhaps the pipeline could be
+used without it*
+
+The image used is specified in repairnator.cfg, and is by default
+```surli/repairnator:latest```.
```
Comment (surli) [inline doc/Dockerpool/SkipDeleteArchitectureNotes.md]: Actually it changed: the default is no more on my personal docker hub repository but on the Spirals Team's one, so it's `spirals/repairnator:latest`

Comment (btellstrom) [reply doc/Dockerpool/SkipDeleteArchitectureNotes.md]: Fixed.

**Comment by btellstrom** [comment]
@surli anything you feel that I have missed?

**Comment by surli** [comment]
@btellstrom I merge it we can always improve it later :) Thanks!

**Pull request merged**